plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadataCompressionCodec.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.iceberg;
+
+import org.apache.iceberg.TableMetadataParser.Codec;
+
+public enum IcebergMetadataCompressionCodec
+{
+    NONE,
+    GZIP,
+    /**/;
+
+    public Codec toIceberg()
+    {
+        return switch (this) {
+            case NONE -> Codec.NONE;
+            case GZIP -> Codec.GZIP;
+            default -> throw new IllegalArgumentException("Unsupported Trino metadata compression codec: " + this);
+        };
+    }
+
+    public static IcebergMetadataCompressionCodec fromIceberg(Codec format)
+    {
+        return switch (format) {
+            case NONE -> NONE;
+            case GZIP -> GZIP;
+            default -> throw new IllegalArgumentException("Unsupported Iceberg metadata compression codec: " + format);
```

Comment:
Remove `default`.
It's not necessary today and will force the code to be updated when Iceberg library changes